assets/css/custom.scss

```
@@ -1951,7 +1951,10 @@ footer {
       }
     }
   }
-  #our-word {
+  // /services/audit mirrors this section (content/sections/iac-our-word.md,
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)